### ISSUE TITLE
Duplicates & severity

### ISSUE THREAD

**Issue by jefrai** [nus-cs2103-AY2223S2/forum#385]
Suppose the same bug was reported 3 times by different testers, but one of them rated it as `severity.Low` while the other two rated it `severity.Medium`.

If a team were to accept the `severity.Low` issue and mark the two `severity.Medium` issues as duplicates of it, would there be any recourse for the two testers if they wanted to stick with their original `severity.Medium` judgement in Phase 3? Could they contest it under _severity downgraded_ or _bug flagged as duplicate_?

Also, if such an objection was accepted, would the three issues be considered as one `severity.Medium` bug for team penalty?

**Comment by damithc** [comment]
@jefrai Yes, all three testers will get a chance to respond independently, if the severity chosen by the team is lower than the one the tester chose.
If any of the testers succeeded in upgrading the severity later, all three testers will benefit from the upgraded severity.
